### DISCUSSION TITLE
Filter issues not in a specific project board

### DISCUSSION THREAD

**Discussion by dantebarba** [community/community#4666] [Projects and Issues]
I'm currently migrating a project-specific board to a organization-level board, so I was trying to filter all the issues that are present in one board but not in the other one. The idea was simple something like: "find all the issues that are not assigned to a project named xxxxx", but I couldn't manage to find a way to do it.

I've tried:

`no:project:myboard
`
`not project:myboard
`

None of these work. I can just use no:project but that's not what I want. I want to specify which project to filter out, so I guess the feature is missing.

**Comment by michael-hawker** [comment]
@dantebarba you can do `-repo:namehere` to filter out a specific repo, but it'd be nice to have an organization level filter. I opened #19096 for that.

**Reply by dantebarba** [reply]
Yeah but it's not filterable like the labels. You cannot click onto the repo name of each card and use the filter like that. It forces our team to use labels (like backend, frontend and mobile) to differentiate the projects just to be able to use quick filtering in the organization board. Typing the project name in the search bar is much slower and prone to errors than just clicking a card.
